05/styles/global.css

```diff
@@ -0,0 +1,25 @@
+.container {
+    margin: 5px auto;
+    width: 98%;
```

Comment:
Tutaj raczej ustawiłbym 100% oraz bez wyśrodkowania bo jak jest 100% to nie ma co środkować :)